api/task.go

```diff
@@ -357,51 +355,89 @@ func (h TaskHandler) Update(ctx *gin.Context) {
 		return
 	}
 
-	h.Status(ctx, http.StatusNoContent)
+	h.Status(ctx, http.StatusAccepted)
+}
+
+// Patch godoc
+// @summary Patch a task.
+// @description Patch a task.
+// @tags tasks
+// @accept json
+// @success 202
+// @router /tasks/{id} [put]
+// @param id path int true "Task ID"
+// @param task body Task true "Task data"
+func (h TaskHandler) Patch(ctx *gin.Context) {
+	id := h.pk(ctx)
+	var m model.Task
+	err := h.DB(ctx).First(&m, id).Error
+	if err != nil {
+		_ = ctx.Error(err)
+		return
+	}
+	r := &Task{}
+	r.With(&m)
+	err = h.Bind(ctx, r)
+	if err != nil {
+		_ = ctx.Error(err)
+		return
+	}
+	rtx := WithContext(ctx)
+	task := &tasking.Task{}
+	task.With(r.Model())
+	task.UpdateUser = h.BaseHandler.CurrentUser(ctx)
+	err = rtx.TaskManager.Update(h.DB(ctx), task)
+	if err != nil {
+		_ = ctx.Error(err)
+		return
+	}
+
+	h.Status(ctx, http.StatusAccepted)
 }
 
 // Submit godoc
 // @summary Submit a task.
-// @description Submit a task.
+// @description Patch and submit a task.
```

Comment:
**hint**:  Much better for the caller to be able to patch a task when submitting rather than update.  This will support submit with minimal or no patch.